classes/checkout/CheckoutAddressesStep.php

```diff
@@ -81,10 +81,12 @@ public function handleRequest(array $requestParams = array())
         }
 
         // Can't really hurt to set the firstname and lastname.
-        $this->addressForm->fillWith(array(
+        $this->addressForm->fillWith(
+            array(
```

Comment:
indentation is broken, could you run php-cs-fixer please?

Reply:
@PierreRambaud it wasn't me :) it was php-cs-fixer.
I'll redo later.